.github/workflows/demo-testing.yml

```
@@ -103,6 +130,19 @@ jobs:
       - uses: actions/checkout@v3
         if: needs.detect-changes.outputs[matrix.provider] == 'true'
 
+      - name: Build java-analyzer-bundle from PR and tag as jdtls-server-base
+        if: matrix.provider == 'java' && needs.detect-changes.outputs.java == 'true' && needs.parse-pr-inputs.outputs.BUILD_BUNDLE == 'true'
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)